KsApi/models/graphql/adapters/CreateBackingEnvelope+CreateBackingMutation.swift

```diff
@@ -0,0 +1,71 @@
+@testable import KsApi
+import XCTest
+
+final class CreateBackingEnvelope_CreateBackingMutation: XCTestCase {
+  func test_SCA() {
```

Comment:
I think it's fine to keep it as SCA because it is a notable term in the payments space. It stands for "Strong Customer Authentication" and it's commonplace specifically in the European Union.